core/market/src/config.rs

```diff
@@ -22,17 +22,18 @@ pub struct DiscoveryConfig {
     #[structopt(env, parse(try_from_str = humantime::parse_duration), default_value = "4min")]
     pub mean_cyclic_unsubscribes_interval: Duration,
     #[structopt(env, parse(try_from_str = humantime::parse_duration), default_value = "5sec")]
-    pub offer_broadcast_time: Duration,
+    pub offer_rebroadcast_delay: Duration,
```

Comment:
its not only re-broadcast, but all broadcasts

+1 for time -> delay

Reply:
done